### ISSUE TITLE
Visually Separate Social Login

### ISSUE THREAD

**Issue by thelouisvivier** [stonith404/pingvin-share#360]
### 🔖 Feature description

You should separate visually the different Oauth providers in the web interface. For example, add space between each, or un straight line. This would be easier to configure and read each Oauth provider.

### 🎤 Pitch

When you go on the Social Login page, it’s quite hard to spot each Oauth provider and it’s parameters. 
It would be better if they were all separated from each other.

**Comment by zz5840** [comment]
I had this feature in mind when I was developing OAuth, but unfortunately the configuration page is now auto-generated.
@stonith404 Do you have any ideas?

**Comment by stonith404** [comment]
@zz5840 Yeah it's a good point. Since OAuth there are now a lot more configuration variables and the configuration page gets confusing. I think it would make sense to hard code the configuration page in the future as this would allow more customization.
